dev-perl/Test-More-UTF8/Test-More-UTF8-0.05.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DIST_AUTHOR=MONS
+DIST_VERSION="0.05"
+
+inherit perl-module
+
+DESCRIPTION="Enhancing Test::More for UTF8-based projects"
+
+SLOT="0"
+LICENSE="|| ( Artistic GPL-1+ )"
```

Comment:
This LICENSE statement is redundant under perl-module.eclass, its provided as the default due to prolific application of packages saying to use "The same license as perl"